src/Sarif.FunctionalTests/SarifConverterTestData/ClangAnalyzer/FileNumberCoverage.xml.sarif

```diff
@@ -5,19 +5,26 @@
       "toolInfo": {
         "name": "Clang"
       },
+      "runInfo": {
+        "fileInfo": {
+          "jcparam.c": [
+            {
+              "uri": "jcparam.c",
+              "mimeType": "text/x-cpp"
+            }
+          ]
+        }
+      },
       "results": [
         {
           "ruleId": "Dead assignment",
           "fullMessage": "Value stored to 'scanptr' is never read",
           "locations": [
             {
-              "analysisTarget": [
-                {
-                  "uri": "jcparam.c",
-                  "mimeType": "text/x-cpp",
-                  "region": {}
-                }
-              ]
+              "analysisTarget": {
+                "uri": "jcparam.c",
+                "region": {}
```

Comment:
ok, back to the never ending nuance around regions. Our spec says that an explicit startLine > 0 shall denote a text region. Our spec does not make it clear that a binary region should have an explicit offset property >=0. that would allow us to detect cases like in this converter example, in which the region isn't populated at all because the converted log file apparently has no corresponding information. alternately, we could require in this case that the converter not emit empty region at all. thoughts? [we should file/fix this as a minor SARIF spec issue]